src/simulator_worker/utils.py

```diff
@@ -105,10 +119,17 @@ def create_output_esdl(
     profiles = ProfileManager()
     profiles.profile_type = "DATETIME_LIST"
     profiles.profile_header = ["datetime"]
+
     for series_name, _ in simulation_result.items():
         logger.debug(f"Output series: {series_name}")
-        # print(f"{series_name}:\t\t {asset.port}")
-        # print(f"Inport index={get_port_index(asset, esdl.InPort)}")
+        asset = _id_to_asset(series_name[0], esh.energy_system)
+        port_index = -1
+        if series_name[1].endswith("supply"):
+            port_index = get_port_index(asset, esdl.InPort)
+        else:
+            port_index = get_port_index(asset, esdl.OutPort)
+        logger.debug(f"{series_name}:\t\t {asset.port}")
+        logger.debug(f"Inport index={get_port_index(asset, esdl.InPort)}")
```

Comment:
The format string of this log line is an f-string. This means that the string gets created regardless if the log line needs to be outputted or not (due to log levels). Should rather use the alternative: `logger.debug("Inport index={}", get_port_index(asset, esdl.InPort))`

Reply:
Couple log lines unchanged. Fixed with https://github.com/Project-OMOTES/simulator-worker/pull/8/commits/bb42a08db6a5a95390a678975d6908182fcb9441